src/main/java/com/github/k0kubun/builder/query/graphql/GraphQLQueryBuilder.java

```diff
@@ -0,0 +1,22 @@
+package com.github.k0kubun.builder.query.graphql;
+
+import com.github.k0kubun.builder.query.graphql.builder.GraphQLBuilderFactory;
+import com.github.k0kubun.builder.query.graphql.builder.ObjectBuilder;
+import com.github.k0kubun.builder.query.graphql.builder.QueryBuilder;
+
+public class GraphQLQueryBuilder
```

Comment:
Renamed this class from GraphQL to GraphQLQueryBuilder so that it does not class with classes from libraries like graphql-java and graphql-java-kickstart